code/modules/mining/fulton.dm

```diff
@@ -35,6 +35,7 @@ var/list/total_extraction_beacons = list()
 		if(!A)
```

Comment:
Done. This has the side effect though, the list pops up even when there is just only one beacon in the word.